src/evo/assetlocktx.h

```diff
@@ -9,6 +9,7 @@
 #include <primitives/transaction.h>
 #include <gsl/pointers.h>
 
+#include <core_io.h>
```

Comment:
could you add here a forward declaration of ScriptPubKeyToUniv or do something else?

To avoid:
```
"core_io -> evo/assetlocktx -> core_io"
```